### ISSUE TITLE
Default Action option missing in Brain->Player->Discrete Action Space

### ISSUE THREAD

**Issue by hsaikia** [Unity-Technologies/ml-agents#1268]
Hi,

The default action under 'Edit the discrete inputs for your actions' is missing in the current main branch. Because of this I am not able to test my agent under the Player Brain. Can anyone confirm if this is true, or am I missing something?

Regards

**Comment by xiaomaogy** [comment]
Hi @hsaikia , thanks for the note. Yes you are correct, we've removed the default action since v0.5. The reason is with discrete action now it is possible that we can have multiple branches, and it would be really verbose if people have to specify multiple default action for each of these branches. Since v0.5, we've made the action index 0 as the default action. Maybe we need to update our migration doc to explain this....

**Comment by hsaikia** [comment]
Hi @xiaomaogy , thanks for the reply. This is fine. I found a workaround for this (which should be possible for everyone else). I indexed my actions starting from 1 and added an extra action. So, for instance, instead of 4 actions I now have 5 actions and they are assigned indices 1 through 4.

**Comment by xiaomaogy** [comment]
@hsaikia Yes, that's the standard way of migrating from v0.4 to v0.5 for discrete control. (Actually we did this during our upgrade for all of our example environments. )

**Comment by araii** [comment]
>  I indexed my actions starting from 1 and added an extra action. So, for instance, with 4 actions I now set aside space for 5 actions and index my actions from 1 through 4 with 0 being the default.

that will work in Player brain type while testing out the inputs...but it fail for when training with External brain type. It messes up the array sequence cos' Tensorflow returns with index of 0, 1, 2...but Player mode starts off with 0 as default, i can't set it to -1. 

actually i don't see how default will affect multiple branches...default is a value specifically for AgentAction to ignore when there are no inputs.

**Comment by araii** [comment]
..ah i see...got it. thanks, i'll just +1 to the indices returned from External brain.
